### ISSUE TITLE
Video-JEPA Moving MNIST: late-stage validation degradation with default effective batch size

### ISSUE THREAD

**Issue by St-Hun** [facebookresearch/eb_jepa#21]
Hi, thank you for releasing EB-JEPA.

I am trying to reproduce the Video-JEPA Moving MNIST example using the provided/default configuration. However, I consistently observe a late-stage failure mode where the training losses continue to decrease or remain low, while the validation losses increase sharply and then saturate.

This looks like severe overfitting, or possibly a representation collapse / degenerate solution.

## Setup

* Repository: facebookresearch/eb_jepa
* Example: `examples/video_jepa`
* Dataset: Moving MNIST
* Config: default Video-JEPA config
* Default effective batch size: 64
* Training rollout steps: `K = 4`
* GPU: RTX 4090 24GB

I tested the following settings:

1. A smaller micro-batch with gradient accumulation, intended to match the default effective batch size of 64.
   The same late-stage validation degradation was observed in this setting.

2. Batch size 16 with all other config values unchanged, mainly to avoid OOM.
   I understand that this is not an exact reproduction of the default batch-size-64 setting, since the optimization dynamics may change. However, I observed a similar degradation pattern in this setting as well.

To keep the issue focused, I attached only the curves from the gradient accumulation run, where the effective batch size was set to 64.

## Observed behavior

In the gradient accumulation setting, I observe the following pattern:

* `train/pred_loss` decreases and remains very low.
* `train/vc_loss` also decreases and remains low.
* However, `val/recon_loss` increases after the early training stage.
* `val/det_loss` also increases sharply and saturates around a high value.

For example, with gradient accumulation and effective batch size 64:

* `train/pred_loss` decreases to a very small value and remains low.
* `train/vc_loss` also remains low.
* `val/recon_loss` increases from around 0.005 to around 0.038.
* `val/det_loss` increases from around 0.03–0.04 to around 0.10.

I attached the following training curves from this run:

* `train/pred_loss`
* `train/vc_loss`
* `val/recon_loss`
* `val/det_loss`

<img width="5056" height="2656" alt="Image" src="https://github.com/user-attachments/assets/f04ed372-55cd-48be-8aed-4572a2dfcdc8" />
<img width="5056" height="2656" alt="Image" src="https://github.com/user-attachments/assets/415a5f0c-f6a5-4dd3-a9cb-ef32bca01750" />
<img width="5056" height="2656" alt="Image" src="https://github.com/user-attachments/assets/a672bf90-7942-4800-aded-9d7a5cd0c90b" />
<img width="5056" height="2656" alt="Image" src="https://github.com/user-attachments/assets/22f6a389-ee9d-445d-b09c-04788f83742f" />

## Expected behavior

Based on the paper/README, I expected the Video-JEPA Moving MNIST example to remain stable under the default configuration, with prediction loss decreasing while downstream validation quality improves or at least does not degrade late in training.

## Question

Is this behavior expected for the current default Video-JEPA implementation?

More specifically:

1. Is the late-stage increase in `val/recon_loss` and `val/det_loss` a known issue?

2. Could this indicate representation collapse or another degenerate solution, even if the variance-covariance regularization loss remains low?

3. Is the Video-JEPA Moving MNIST result expected to require a physical batch size of 64, or should gradient accumulation with the same effective batch size also be valid?

4. Could BatchNorm or the variance-covariance regularizer make the physical micro-batch size important in this example?

5. Are there recommended settings for Moving MNIST, especially under limited GPU memory or smaller physical batch sizes, such as:

   * early stopping,
   * different `std_coeff` / `cov_coeff`,
   * lower learning rate,
   * disabling or changing gradient accumulation,
   * changing BatchNorm-related behavior when using gradient accumulation,
   * using SIGReg instead of the variance-covariance regularizer for smaller physical batch sizes?

6. Is SIGReg expected to be more stable than the variance-covariance regularizer in this Video-JEPA setting when the physical batch size is small, or is VICReg/VC regularization still the recommended default for Moving MNIST?

7. Is the paper’s reported Video-JEPA result obtained with exactly the current default config, or were there additional settings or checkpoint selection criteria?

Any guidance would be appreciated. Thank you.
